test/integration/smartCache.js

```diff
@@ -0,0 +1,28 @@
+const app = require('../../server');
+const constants = require('../../app/lib/constants');
+const chai = require('chai');
+const chaiHttp = require('chai-http');
+
+const expect = chai.expect;
+
+chai.use(chaiHttp);
+
+describe('smartCache', () => {
+  it('should set cache-control header for valid page', (done) => {
```

Comment:
I know the value of `CACHE_TIMEOUT_SECONDS` is implicitly tested here but it would be good to have that included in the description of the test.